app/models/client.rb

```diff
@@ -29,6 +29,8 @@ class Client < ApplicationRecord
   has_many :notes
   has_many :system_notes
   has_many :tax_returns
+  accepts_nested_attributes_for :tax_returns
```

Comment:
Facilitates updating the tax returns and intake through the client on the client forms, avoiding multiple query updates and transactions in the form objects.

Reply:
oh nice! we can pass it to the parent model!